libs/barista-components/radial-chart/src/utils/radial-chart-utils.ts

```diff
@@ -85,3 +85,64 @@ export function generatePathData(
     endAngle,
   });
 }
+
+/**
+ * Returns an array of percentages with decimal precision.
+ */
+export function getPercentages(
```

Comment:
We could move this to the percentage formatter or to the core. 
@tomheller What do you think?

Reply:
For now, it looks good! I suggest a follow-up issue if we want to move the functions

Reply:
Thank you @rowa-audil, as I comment in the [Issue](https://github.com/dynatrace-oss/barista/issues/1959), we might have this behavior for all charts whose percentages must sum up 100. I am open to move this function to an upper level, so that other components can use it as well.